src/ucp/core/ucp_worker.c

```diff
@@ -2912,6 +2916,10 @@ ucp_worker_do_keepalive_progress(ucp_worker_h worker)
         ucp_worker_keepalive_next_ep(worker);
     }
 
+    ka_ep_count     = worker->keepalive.ep_count;
```

Comment:
won't it be simpler just init 0 0 and ++ every iteration?

Reply:
tyes, makes sense